whois-api/src/main/java/net/ripe/db/whois/api/rest/search/ManagedAttributeSearch.java

```diff
@@ -17,24 +16,25 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.dao.DataAccessException;
 import org.springframework.stereotype.Component;
 
 import javax.annotation.Nullable;
-import java.util.Optional;
-import java.util.stream.Stream;
+
+import static net.ripe.db.whois.common.rpsl.AttributeType.ORG;
```

Comment:
Inconsistent imports (some fields are imported, some are not)

Reply:
Done.